From dbfd8efea1a2c7ea5ba1bcab01329440b58c2263 Mon Sep 17 00:00:00 2001
From: Ke Qinghua <qinghua.ke@freescale.com>
Date: Fri, 9 May 2014 13:30:13 +0800
Subject: [PATCH 318/602] ENGR00293774 Geomagnetic sensor doesn't work after
 encryption. 100%

Change magd service class from main to late_start

Signed-off-by: Ke Qinghua <qinghua.ke@freescale.com>
---
 sabreauto_6q/init.rc | 2 +-
 sabresd_6dq/init.rc  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index 7d312cc..91ae165 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -139,7 +139,7 @@ service wpa_supplicant /system/bin/wpa_supplicant \
 
 # magd daemon
 service magd /system/bin/magd
-    class main
+    class late_start
     user system
     group input
     oneshot
diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index 9e9f555..79d6ccd 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -156,7 +156,7 @@ service wpa_supplicant /system/bin/wpa_supplicant \
 
 # magd daemon
 service magd /system/bin/magd
-    class main
+    class late_start
     user system
     group input
     oneshot
-- 
1.8.0

